### ISSUE TITLE
appendTo() can somehow sort-of lose track of the element in question?

### ISSUE THREAD

**Issue by alanhogan** [madrobby/zepto#625]
In this case, height is zero:

```
  $source.detach().appendTo($unit)
  window.console?.log? "AFTER DETACH & APPEND, height is: #{$source.height()}"
```

But in this case, height is as expected:

```
  srcID = $source[0].id
  $source.detach().appendTo($unit)
  $source = $("##{srcID}") if window.Zepto
  window.console?.log? "AFTER DETACH & APPEND, height is: #{$source.height()}"
```

(examples in coffeescript)

note `.detach` is defined [as it is here](https://gist.github.com/3935463), basically a .clone(), remove, return deal.
